From: Robert Luberda <robert@debian.org>
Date: Wed, 16 Apr 2014 09:14:47 +0200
Subject: 11-nfsiostat-whatis-entry

Fix `manpage-has-bad-whatis-entry usr/share/man/man1/nfsiostat-sysstat.1.gz'
lintian warning.
---
 man/nfsiostat-sysstat.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/nfsiostat-sysstat.in b/man/nfsiostat-sysstat.in
index c8a4ed7..7d4bed9 100644
--- a/man/nfsiostat-sysstat.in
+++ b/man/nfsiostat-sysstat.in
@@ -1,6 +1,6 @@
 .TH NFSIOSTAT-SYSSTAT 1 "JANUARY 2014" Linux "Linux User's Manual" -*- nroff -*-
 .SH NAME
-nfsiostat-sysstat (the nfsiostat command from the sysstat package) \- Report input/output statistics for network filesystems (NFS).
+nfsiostat-sysstat \- (the nfsiostat command from the sysstat package) Report input/output statistics for network filesystems (NFS).
 .SH SYNOPSIS
 .ie 'yes'@WITH_DEBUG@' \{
 .B nfsiostat-sysstat [ -h ] [ -k | -m ] [ -t ] [ -V ] [ --debuginfo ] [
